tests/examples/pacer/nef/s3/paeb_3.json

```diff
@@ -16,7 +16,7 @@
           "pacer_doc_id": null,
           "pacer_magic_num": null,
           "pacer_seq_no": null,
-          "short_description": ""
+          "short_description": "Close Adversary Case - Tri"
```

Comment:
I'm not sure this looks right. Probably we need to check this on the document history report. If you find the case on CL, you can find a link to that report: 

![image](https://github.com/freelawproject/juriscraper/assets/236970/1319eaa8-962f-4992-8ef4-a7e1b5103ae6)